tests/Oryx.Integration.Tests/Python/PythonDynamicInstallationTest.cs

```diff
@@ -65,6 +65,50 @@ public async Task CanBuildAndRunPythonApp(string pythonVersion)
                 });
         }
 
+        [Theory]
+        [InlineData("2.7")]
```

Comment:
user constants from constant file instead of hardcoded versions?

Reply:
Sure